lib/ui/profile/interactor/profile_bloc.dart

```diff
@@ -74,13 +74,20 @@ class ProfileBloc extends Bloc<ProfileEvent, ProfileState> {
   }
 
   FutureOr<void> _setName(_SetName event, Emitter<ProfileState> emit) async {
-    emit(state.copyWith(pageState: PageState.loading));
+    emit(state.copyWith(showUpdateBioLoading: true));
     final result = await _setNameUseCase.run(event.name);

```

Comment:
any reason it is using showUpdateBioLoading when updating name? 

Should this renamed to "showUpdateLoading"?